pwiz_tools/Skyline/Jamfile.jam

```diff
@@ -343,15 +343,15 @@ if [ modules.peek : NT ] && --i-agree-to-the-vendor-licenses in [ modules.peek :
         ) > "$(<)"
     }
 
+    constant SKYLINE_PROSIT_CONFIG : "$(SKYLINE_PATH)/Model/Prosit/Config/PrositConfig_production.xml" ;

```

Comment:
I might be misremembering how constants work in .jam files, but I kind of remember that you are only allowed to assign a constant a value once.

If you find that this build fails in "PrositConfig_production.xml" is not there, then you might need to change this to:
```
    constant SKYLINE_PROSIT_CONFIG_PRODUCTION : "$(SKYLINE_PATH)/Model/Prosit/Config/PrositConfig_production.xml" ;
    if [ path.exists $(SKYLINE_PROSIT_CONFIG_PRODUCTION) ] 
    {
        constant SKYLINE_PROSIT_CONFIG : $(SKYLINE_PROSIT_CONFIG_PRODUCTION) ;
    }
    else if --official in [ modules.peek : ARGV ]
    {
        errors.user-error "PrositConfig_production.xml must be present for an official build" ;
    }
    else
    {
        constant SKYLINE_PROSIT_CONFIG : "$(SKYLINE_PATH)/Model/Prosit/Config/PrositConfig_development.xml" ;
    }
```

But, if the build works correctly when "PrositConfig_production.xml" is not there, then it means I was wrong.
(I have tried to test this on my own computer and am just getting myself more confused)

Reply:
I didn't remember myself whether constant works that way but it was easy enough to test:
```
constant foo : "Constants are really constant" ;
constant foo : "Constants can be changed" ;
echo $(foo) ;
```

Echoes "Constants can be changed".